CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## [1.0.0] - 2021-09-17
```

Comment:
It's not September yet 😀

Reply:
Hah! My typing skills on display, I see.